target/linux/ramips/dts/TL-MR3020V3.dts

```diff
@@ -0,0 +1,141 @@
+/dts-v1/;
+
+#include "mt7628an.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "tplink,tl-mr3020-v3", "mediatek,mt7628an-soc";
+	model = "TP-Link TL-MR3020 v3";
+
+	chosen {
+		bootargs = "console=ttyS0,115200";
+	};
+
+	memory@0 {
+		device_type = "memory";
+		reg = <0x0 0x4000000>;
+	};
+
+	gpio-keys-polled {
+		compatible = "gpio-keys-polled";
+		#address-cells = <1>;
+		#size-cells = <0>;
+		poll-interval = <20>;
+
+		wps {
+			label = "wps";
+			gpios = <&gpio1 6 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_WPS_BUTTON>;
+		};
+
+		modec1 {
+			label = "sw1";
+			gpios = <&gpio1 9 GPIO_ACTIVE_LOW>;
+			linux,code = <BTN_0>;
+		};
+
+		modec2 {
+			label = "sw2";
+			gpios = <&gpio1 10 GPIO_ACTIVE_LOW>;
+			linux,code = <BTN_1>;
+		};
+	};
+
+	gpio-leds {
+		compatible = "gpio-leds";
+
+		power {
+			label = "tp-link:green:power";
+			gpios = <&gpio1 5 GPIO_ACTIVE_LOW>;
+			default-state = "on";
+		};
+
+		wan {
+			label = "tp-link:green:3g";
+			gpios = <&gpio1 11 GPIO_ACTIVE_LOW>;
+		};
+
+		wlan {
+			label = "tp-link:green:wlan";
+			gpios = <&gpio1 12 GPIO_ACTIVE_LOW>;
+		};
+
+		wps {
+			label = "tp-link:green:wps";
+			gpios = <&gpio0 2 GPIO_ACTIVE_LOW>;
+			linux,default-trigger = "timer";
+		};
+
+		lan {
+			label = "tp-link:green:lan";
+			gpios = <&gpio0 3 GPIO_ACTIVE_LOW>;
+		};
+
+};
+};
+
+&spi0 {
+	status = "okay";
+
+	m25p80@0 {
+	#address-cells = <1>;
```

Comment:
address and size cells aren't required here.

Please fix the indention for this block as well.